packages/next/server/lib/find-page-file.ts

```diff
@@ -1,8 +1,12 @@
 import { join } from 'path'
 import {isWriteable} from '../../build/is-writeable'
 
-export async function findPageFile(rootDir: string, normalizedPagePath: string, pageExtensions: string[]): Promise<string|null> {
-  for (const extension of pageExtensions) {
+export async function findPageFile(rootDir: string, normalizedPagePath: string, pageExtensions: string[], amp: boolean): Promise<string|null> {
+  // Add falling back to .amp.js extension
+  if (!amp) pageExtensions = pageExtensions.concat('amp.js')
```

Comment:
This needs to work with all `pageExtensions`, not just `.js`.

Reply:
Good catch